cc/src/main/scala/me/sgrouples/rogue/cc/MongoAsyncBsonJavaDriverAdapter.scala

```diff
@@ -177,9 +177,13 @@ class BatchingCallback[R, T](r: RogueBsonRead[R], f: Seq[R] => Future[Seq[T]])(i
       } else if (docs == null) {
         p.success(resBuilder.result())
```

Comment:
I wander if I should not close batchCursor here as well. I am trying to find if executor closes it